admin/qtx_admin.php

```diff
@@ -47,6 +47,11 @@ function qtranxf_collect_translations( &$qfields, &$request, $edit_lang ) {
         if ( ! qtranxf_isMultilingual( $request ) ) {
             // convert to ML value
             $qfields[ $edit_lang ] = $request;
+            //Handles same translation for all languages to overwrite properly previous translations (change ignored otherwise)
+            if ( ! is_null( qtranxf_allthesame( $qfields ) ) ) {
+                array_walk( $qfields, function( &$value ){ $value=""; } );
+                $qfields[qtranxf_getLanguageDefault()] = $request;
+            }
```

Comment:
When all the languages are the same we don't want to store ML tags, it should be stored as a simple string.
This works fine for post titles, so the bug is specific to terms. That part of code should not change.

Reply:
agree, reverted previous commit and added term names specific fix in https://github.com/qtranslate/qtranslate-xt/pull/1230/commits/2a1486fd4dcb25627795730509c2944887b55a66